sentry.client.config.ts

```diff
@@ -49,7 +49,8 @@ Sentry.init({
 function getEnvironment() {
   if (
     document.location.origin === "https://monitor.firefox.com" ||
-    document.location.origin === "https://monitor.mozilla.com"
+    document.location.origin === "https://monitor.mozilla.com" ||
+    document.location.origin === "https://monitor.mozilla.org"
```

Comment:
why `.com` _and_ `.org` TLDs? 

- https://vpn.mozilla.org/ redirects to https://www.mozilla.org/en-US/products/vpn/
- https://vpn.mozilla.com/ seems to just fail outright.

Reply:
`.com` was already in-place, I just added `||` - I wanted to make this PR as additive as possible. If we don't end up using `.com` we can remove.